pyvcloud/vcd/org.py

```diff
@@ -118,6 +118,27 @@ def get_catalog(self, name):
                 return self.client.get_resource(link.href)
         raise Exception('Catalog not found.')
 
+    def get_user(self, user_name):
+        if self.resource is None:
+            self.resource = self.client.get_resource(self.href)
+        resource_type = 'adminUser' if self.is_admin else 'user'
+        org_filter = 'org==%s' % self.resource.get('href') if self.is_admin else None
```

Comment:
Run flake8 and make sure formatting is correct

pip install -r test-requirements.txt
flake8 pyvcloud/vcd

pip install -r test-requirementes.txt
flake8 vcd_cli/*.py